packages/lockfile-lint-api/src/validators/ValidateUrl.js

```diff
@@ -0,0 +1,52 @@
+'use strict'
+
+const {isAllowedUrl, isValidHost} = require('../common/utils')
```

Comment:
ValidatorUrl should really only work with just Urls, and not be aware of hosts and whether the host is valid or not.

Reply:
Removed this